File: QuantConnect.AlphaStream.Python/Models/RuntimeError.py
from datetime import datetime


class RuntimeError:
    """Individual error set for an Alpha in the QuantConnect Alpha Streams market"""

    def __init__(self, json):
        self.Time = datetime.utcfromtimestamp(json['time']) if 'time' in json else None

        self.Error = json.get('error', None)

        self.Stacktrace = json.get('stacktrace', None)

    def __repr__(self):
        error = '' if self.Error is None else self.Error[:10]
        stacktrace = '' if self.Stacktrace is None else self.Stacktrace[:10]
        return f'Error at {self.Time}, message: {error}, stacktrace: {stacktrace}'

File: QuantConnect.AlphaStream/Models/RuntimeError.cs
using Newtonsoft.Json;
using System;
using QuantConnect.Util;

namespace QuantConnect.AlphaStream.Models
{
    /// <summary>
    /// Individual error set for an Alpha in the QuantConnect Alpha Streams market
    /// </summary>
    public class RuntimeError
    {
        /// <summary>
        /// The unix timestamp of the production runtime error.
        /// </summary>
        [JsonProperty("time"), JsonConverter(typeof(DoubleUnixSecondsDateTimeJsonConverter))]
        public DateTime Time { get; set; }

        /// <summary>
        /// Error message string from the Alpha.
        /// </summary>
        [JsonProperty("error")]
        public string Error { get; set; }

        /// <summary>
        /// Stacktrace of the production error recorded from the Alpha.
        /// </summary>
        [JsonProperty("stacktrace")]
        public string StackTrace { get; set; }
    }
}